src/components/AppSidebar/AppSidebar.vue

```diff
@@ -164,7 +164,7 @@
 				</div>
 			</header>
 
-			<AppSidebarTabs>
+			<AppSidebarTabs ref="tabs">
```

Comment:
Is this ref used somewhere?

Reply:
It will. I'd rather have it in the meantime

Reply:
@raimund-schluessler this PR is a scam https://github.com/nextcloud/nextcloud-vue/pull/1303#discussion_r472010825

Reply:
Who knows what @skjnldsv is really up to.